_books/ion-1-1/src/binary/e_expressions.md

```diff
@@ -120,326 +136,529 @@ EF 01
+|----------|--------------------|:-------------:|-----------------------------------------------------------------------------------------------------------------------|
+| `int`    | `uint8`            |       1       | [`FixedUInt`](primitives/fixed_uint.md)                                                                               |
+|          | `uint16`           |       2       |                                                                                                                       |
+|          | `uint32`           |       4       |                                                                                                                       |
+|          | `uint64`           |       8       |                                                                                                                       |
+|          | `flex_uint`        |   variable    | [`FlexUInt`](primitives/flex_uint.md)                                                                                 |
+|          | `int8`             |       1       | [`FixedInt`](primitives/fixed_int.md)                                                                                 |
+|          | `int16`            |       2       |                                                                                                                       |
+|          | `int32`            |       4       |                                                                                                                       |
+|          | `int64`            |       8       |                                                                                                                       |
+|          | `flex_int`         |   variable    | [`FlexInt`](primitives/flex_int.md)                                                                                   |
+| `float`  | `float16`          |       2       | [Little-endian IEEE-754 half-precision float](https://en.wikipedia.org/wiki/Half-precision_floating-point_format)     |
+|          | `float32`          |       4       | [Little-endian IEEE-754 single-precision float](https://en.wikipedia.org/wiki/Single-precision_floating-point_format) |
+|          | `float64`          |       8       | [Little-endian IEEE-754 double-precision float](https://en.wikipedia.org/wiki/Single-precision_floating-point_format) |
+| `symbol` | `flex_sym`         |   variable    | [`FlexSym`](primitives/flex_sym.md)                                                                                   |
+
+#### Example encoding of an e-expression with primitive, `exactly-one` arguments
+
+As first demonstrated in _[Encoding multiple exactly-one arguments](#encoding-multiple-exactly-one-arguments)_,
+the bytes of the serialized arguments begin immediately after the opcode and (if separate) the macro address.
+The reader iterates over the parameters declared in the macro signature from left to right. For each parameter,
```

Comment:
Sure, I'll use that.

> and if you translate the specification to a language that reads in a different direction, it would still be clear

I'm not sure I follow. If the specification were in, say, Arabic, that wouldn't affect the direction that one would read the parameter names in the macro signature. They would still be left-to-right.